tests/_test_msui/test_remotesensing.py

```diff
@@ -34,10 +42,238 @@ def test_skyfield_data_expiration(recwarn):
+                                                      (80.01270397128556, 26.470217219393053),
+                                                      (79.99072113824144, 26.55675386908735),
+                                                      (79.96871062506634, 26.643287306073162),
+                                                      (79.94667226947563, 26.729817512903203),
+                                                      (79.92460590846814, 26.816344472080264),
+                                                      (79.90251137832155, 26.902868166052826),
+                                                      (79.88038851458653, 26.989388577217603),
+                                                      (79.8582371520822, 27.075905687915036),
+                                                      (79.83605712488925, 27.162419480434753),
+                                                      (79.81384826634594, 27.2489299370091),
+                                                      (79.7916104090419, 27.335437039815744),
+                                                      (79.76934338481207, 27.421940770978352),
+                                                      (79.74704702473245, 27.508441112561886),
+                                                      (79.72472115911295, 27.59493804657693),
+                                                      (79.70236561749279, 27.681431554975404),
+                                                      (79.67998022863429, 27.76792161965211),
+                                                      (79.65756482051724, 27.854408222443364),
+                                                      (79.63511922033271, 27.94089134512787),
+                                                      (79.6126432544781, 28.027370969422904),
+                                                      (79.59013674854992, 28.113847076988744),
+                                                      (79.56759952733928, 28.20031964942261),
+                                                      (79.54503141482452, 28.286788668262982),
+                                                      (79.52243223416616, 28.37325411498613),
+                                                      (79.49980180770042, 28.459715971006098),
+                                                      (79.47713995693299, 28.546174217675254),
+                                                      (79.45444650253313, 28.632628836281985),
+                                                      (79.43172126432714, 28.71907980805177),
+                                                      (79.40896406129221, 28.80552711414579),
+                                                      (79.38617471155024, 28.89197073566029),
+                                                      (79.3633530323609, 28.978410653627137)]
+        self.result_test_tangent_point_coordinates = [(round(x, 2), round(y, 2)) for x, y in
```

Comment:
I'd think it is reasonable to store the data only with two significant digits. This is very cluttered and suggests an unneeded precision.